README.md

```diff
@@ -70,53 +304,184 @@ Make sure you have Postgres, Redis and Vault properly installed & configured. Do
+```
+
+Take this JSON data, copy, and paste into [https://qr.io](https://qr.io).
+
+!["QR.io"](/docs/qrio.png)
 
-In `.env-api`:
+With the Polygon ID app, open it up and scan the QR code.
 
-- `ISSUER_API_UI_AUTH_USER`
-- `ISSUER_API_UI_AUTH_PASSWORD`
-- `ISSUER_API_UI_ISSUER_DID` - obtained when following the steps in [creating the issuer DID](#creating-the-issuer-did).
-- `ISSUER_ETHEREUM_URL` - this is the URL of the issuer's DApp.
-- `ISSUER_API_UI_ISSUER_LOGO` - optional (placeholder used if left blank). A valid URL to a minimum 40x40 pixel PNG, JPEG or SVG of the issuer's logo.
+!["Polygon ID App Adding Claim/Credential"](/docs/polygonid-app-claim.png)
 
-In `.env-issuer`:
+### Verifying Claim
 
-- `ISSUER_API_AUTH_USER`
-- `ISSUER_API_AUTH_PASSWORD`
-- `ISSUER_KEY_STORE_TOKEN` - obtained when following the steps in [adding Ethereum private key to the Vault](#adding-ethereum-private-key-to-the-vault).
+ℹ️ **NOTE:** The goal is to build your own type of claims and ways to verify claims, but this is an example of how things could work.
 
-If you are running the UI, in `.env-ui`:
+A quick way to validate this KYCAge Claim is to use [https://verifier-demo.polygonid.me/](https://verifier-demo.polygonid.me/).
 
-- `ISSUER_UI_AUTH_USERNAME`
-- `ISSUER_UI_AUTH_PASSWORD`
+!["Verifier Selecting KYCAgeCredential"](/docs/verifier-kycagecredential.png)
 
-### Adding Ethereum private key to the Vault
+!["Verifier Verification Prompt"](/docs/verifier-verification.png)
 
-This is required for signing on-chain transactions. In a basic use-case this can be retrieved from an Ethereum wallet that can connect to Polygon Mumbai Testnet.
+!["Polygon ID App Generating Proof"](/docs/polygonid-app-proof.png)
 
-Follow these steps:
+!["Verifier Proof Verified"](/docs/verifier-success-verified.png)
 
-1. Copy your Ethereum private key, pasting it into `<private_key>` in the next step.
-2. Run `make private_key=<private_key> add-private-key`
-3. Run `make add-vault-token`
+---
+## How To Run The Server
 
-### Creating the issuer DID
+### Running the admin server backend (api) for evaluation purposes with docker-composer
 
-This determines the owner of the credentials that are issued. You can either reuse an existing DID already configured, or you can generate a new identity running:`make generate-issuer-did` (or `generate-issuer-did-arm`) and a new issuer did must be in the environment variable `ISSUER_API_UI_ISSUER_DID` in `.env-api`
+1) Configure the project creating config.toml file.
```

Comment:
we should encourage the users to use the .env files instead the config.toml

Reply:
Thanks for the catch.